configs/openmrs_config/ampathforms/NCD_Medical_History.json

```diff
@@ -23,10 +23,22 @@
                                 "concept": "fd588078-9286-46ea-b133-36aab1b025ec",
                                 "answers": [
                                     {
-                                        "concept": "37811074-14d1-41f3-bda9-e3422f6e22b0"
+                                        "concept": ""
                                     },
                                     {
-                                        "concept": "989a3c74-141f-4dcf-9716-d4d3e2eaacdb"
+                                        "concept": ""
+                                    },
+                                    {
+                                        "concept": ""
+                                    },
+                                    {
+                                        "concept": ""
+                                    },
+                                    {
+                                        "concept": ""
+                                    },
+                                    {
+                                        "concept": ""
```

Comment:
@kazlaw could you advise on how to get the concepts for the set of answers(`Local Authority, HC, RH, NGO, Self presented, Other`)? I have looked out in our dictionary manager but I haven't found them.

Reply:
@kdaud, take the example of the Concept `Education` [here](https://github.com/openmrs/ozone-distro-cambodia/blob/main/configs/openmrs_config/concepts/education.csv) notice the Concept is at the bottom of the csv table, with the Data class `Question` and Data type `Coded`. For this case we shall not create a new Concept csv file! Using the example of `Education`  add the Concept in [this ](https://github.com/openmrs/ozone-distro-cambodia/blob/main/configs/openmrs_config/concepts/general.csv) file with the answers above it .The translation is in the Google doc .The Uuids are generated from [here ](https://www.uuidgenerator.net/) under Version 4 UUID Generator

Reply:
Let me know how it pans out .